lldb/test/API/lang/cpp/dwim-print-fixit/TestCppDWIMPrintFixIt.py

```diff
@@ -0,0 +1,27 @@
+"""
+Tests whether the do-what-I-mean (DWIM) print `po` alias applies FixIts like `expr` does
+"""
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class CPP_DWIM_Fixit_TestCase(TestBase):
+    def test_with_run_command(self):
+        "Confirm that the `po` command (alias) applies a FixIt " \
+        "and prints it out to the console, " \
+        "just like the `expression` command."
+        
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "// break here", lldb.SBFileSpec("main.cpp")
+        )
+
+        self.expect(
+            "po class C { int i; void f() { []() { ++i; }(); } }",
+            error = True,
+            substrs=["Applying Fix-It to expression",
+                "changing it to:",
+                "class C { int i; void f() { [this]() { ++i; }(); } }"],
```

Comment:
maybe add `; 1` at the end, so it's not an error?

Reply:
I tried it but the unit test still thinks there's an error, so I have to leave `error = True` in there until we figure that one out.